src/tests/functional/shared_test_classes/src/single_layer/activation.cpp

```diff
@@ -34,7 +34,7 @@ void ActivationLayerTest::SetUp() {
     InferenceEngine::Precision netPrecision;
     std::pair<std::vector<size_t>, std::vector<size_t>> shapes;
     std::pair<ngraph::helpers::ActivationTypes, std::vector<float>> activationDecl;
-    std::tie(activationDecl, netPrecision, inPrc, outPrc, inLayout, outLayout, shapes, targetDevice) = GetParam();
+    std::tie(activationDecl, netPrecision, inPrc, outPrc.front(), inLayout, outLayout, shapes, targetDevice) = GetParam();
```

Comment:
hi @iefode , the parameters in `std::tie` are bounded with `activationParams` in
https://github.com/openvinotoolkit/openvino/blob/a9ff10b365fa8d54f186460d9528c8f27a3ce6c1/src/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/activation.hpp#L86
so just have another check, should we update header files such as `activation.hpp`?

Reply:
Most cases are not changed to vector type in instantiation. It needs to revise the parameter definition and details in test cases. This will cause a lot of changes. If it has to be done, we are going to fix them at final stage.